packages/template-cli/templates/build/azDevops/azure/azure-pipelines-ssr-new-k8s.yml

```diff
@@ -54,6 +54,10 @@ variables:
   pool_vm_image: ubuntu-18.04
   # Test setup
   deployed_base_url: https://stacks-webapp.com/foo
+  # TestCafe E2E Tests
+  testcafe_e2e_test: true
+  # Lighthouse Audit
+  lighthouse_audit: true
```

Comment:
we don't need to be defining variables and their  values here, unless you are passing them down to the method as references to this variable value, e.g: 
```
testcafe_e2e_test: $(testcafe_e2e_test)
```

Reply:
`# Functional tests running on deployed webapp
                  - ${{ if eq(variables.testcafe_e2e_test, true) }}:`